client/router_update.go

```diff
@@ -419,11 +419,11 @@ func (cli *VanClient) RouterUpdateVersionInNamespace(ctx context.Context, hup bo
 		updateRouter = true
 	}
 	if renamedSkRouter {
-		// Updating QDROUTERD_CONF env var
-		envQdrouterdConf := kube.GetEnvVarForDeployment(router, "QDROUTERD_CONF")
-		envQdrouterdConf = strings.ReplaceAll(envQdrouterdConf, "qpid-dispatch", "skupper-router")
-		envQdrouterdConf = strings.ReplaceAll(envQdrouterdConf, "qdrouterd", "skrouterd")
-		kube.SetEnvVarForDeployment(router, "QDROUTERD_CONF", envQdrouterdConf)
+		// Updating SKROUTERD_CONF env var
```

Comment:
Actually we need to rename the old QDROUTERD_CONF to SKROUTERD_CONF here.
The same renaming must be done as part of it for all the other renamed variables.